inginious/frontend/templates/course_admin/submissions.html

```diff
@@ -42,7 +42,7 @@ <h4>$:_('Submissions for task "{0}" - {1} ({2})').format(task_name, complete_nam
             $if str(c['_id']) in old_input['aggregation']:
                 <h4>$:_('Submissions for the task "{0}" - {1}').format(task_name, c["description"])</h4>
 
-<form id="select_form" method="post">
+<form id="select_form" method="get">
```

Comment:
URL have size limitation, but the form can content all the usernames,... I think there is no alternative to POST the form.

Reply:
Yes I wanted to use the GET to enables the bookmarking of options in the page and to avoid inconsistency between the use of GET method when using other links in INGInious like 
"/admin/CS1-Java/submissions?user=bon&task=m01Q11" and the POST method when we perform queries.
I just resolve the inconsistency so, ok for the POST only.